dll/shellext/stobject/power.cpp

```diff
@@ -107,7 +110,25 @@ static HICON DynamicLoadIcon(HINSTANCE hinst)
     {
         index = Quantize(PowerStatus.BatteryLifePercent);
         hBatIcon = LoadIcon(hinst, MAKEINTRESOURCE(br_icons[index]));
-        g_strTooltip.Format(IDS_PWR_PERCENT_REMAINING, PowerStatus.BatteryLifePercent);
+
+        if (PowerStatus.BatteryLifeTime != BATTERY_UNKNOWN_TIME)
+        {
+            uiHour = PowerStatus.BatteryLifeTime / HOUR_IN_SECS;
+            uiMin = (PowerStatus.BatteryLifeTime % HOUR_IN_SECS) / MIN_IN_SECS;
```

Comment:
Side-note: I've noticed that the CRT has a function `div` (or `ldiv` that can return both the quotient and remainder of integer divisions:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/div?view=msvc-150
I'm mentioning this just in case it could be thought to be useful in the future.